.github/ISSUE_TEMPLATE/report-package-problems.yml

```diff
@@ -2,23 +2,36 @@
 name: Packaging problem report
 description: Report problems with a package. This includes build failures or other issues related to a package. This is *not* for bugs with installed software.
```

Comment:
I think make it more specific would be better. "Report problems with **building** a package".

Reply:
That's not the only use of the form, as mentioned in the description. It can also be used to report metadata problems, missing dependencies (I added this text as a clarification) and other packaging related issues.